addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js

```diff
@@ -97,6 +115,24 @@ export default class CustomSubsidyFormFieldsEstimatedCostTableEditComponent exte
                                           this.storeOptions.sourceGraph).length > 0;
   }
 
+  get isAanvraagStep() {
+    const results = this.storeOptions.store.match(
+      undefined,
+      optionsPredicate,
+      undefined,
+      undefined);
```

Comment:
We can use `this.args.field.options` instead (as discovered while chatting ;))

Reply:
Changed it to use this.args, also testing if this.args.field && this.args.field.options exist just to be sure